README.md

```diff
@@ -1,5 +1,3 @@
-# DIY Ads
+# Custom topic list ads
 
-**Theme Summary**
-
-For more information, please see: **url to meta topic**
+Create custom text ads to display on the Discourse topic list
```

Comment:
This might be a good place to capture that this is dependent on the following installed:

```html
<script defer data-domain="meta.discourse.org" src="https://www.discourse.org/js/script.outbound-links.tagged-events.js"></script>
```